etl2/app/mapping/mapping.py

```diff
@@ -42,143 +26,54 @@ def _apply_column_alias(self, df):
-            [
-                "column_name",
-                "casrec_table",
-                "casrec_column_name",
-                "requires_transformation",
-            ]
-        ]
-
-        transformations.set_index("column_name", drop=True, inplace=True)
-        transformations_dict = transformations.to_dict(orient="index")
-
-        t = {}
-        for k, v in transformations_dict.items():
-            transforamtion_name = v["requires_transformation"]
-            original_columns_as_list = v["casrec_column_name"].split(",")
-            cols_to_transform = {
-                "original_columns": [x.strip() for x in original_columns_as_list],
-                "aggregate_col": k,
-            }
-
-            if transforamtion_name in t:
-                t[transforamtion_name].append(cols_to_transform)
-            else:
-                t[transforamtion_name] = [cols_to_transform]
-
-        return t
-
-    def required_columns(self, full_mapping):
-        required_columns = full_mapping.fillna("unknown")
-        required_columns = required_columns[required_columns["is_pk"] != True]
-        required_columns = required_columns[required_columns["nullable"] == False]
-        required_columns = required_columns[required_columns["autoincrement"] == False]
-        required_columns = required_columns[
-            required_columns["requires_transformation"] == "unknown"
-        ]
-        required_columns = required_columns[
-            required_columns["casrec_column_name"] == "unknown"
-        ]
-        required_columns = required_columns[required_columns["default"] == "unknown"]
-        required_columns = required_columns[
-            ["column_name", "data_type", "default_value"]
-        ]
-        required_columns.set_index("column_name", drop=True, inplace=True)
-        required_columns_dict = required_columns.to_dict(orient="index")
-
-        return required_columns_dict
-
     def generate_mapping_df(self):
         raw_mapping_doc = self._read_mapping_doc()
         mapping_doc_with_alias = self._apply_column_alias(df=raw_mapping_doc)
-        mapping_doc_with_unique_id = self._apply_source_unique_id(
-            df=mapping_doc_with_alias
-        )
 
-        return mapping_doc_with_unique_id
+        return mapping_doc_with_alias
 
     def mapping_definitions(self):
 
         mapping_df = self.generate_mapping_df()
 
-        mapping_definitions = {}
-        mapping_definitions["simple_mapping"] = self.simple_mapping(
-            full_mapping=mapping_df
-        )
-        mapping_definitions["transformations"] = self.transformation_mapping(
-            full_mapping=mapping_df
-        )
-        mapping_definitions["required_columns"] = self.required_columns(
-            full_mapping=mapping_df
+        interesting_columns = [
+            "casrec_table",
+            "casrec_column_name",
+            "alias",
+            "requires_transformation",
+            "default_value",
+            # 'is_pk',
```

Comment:
these supposed to be here?

Reply:
They are leftover from me pruning out the next bit. They will appear again...